src/layouts/components/Header.tsx

```diff
@@ -235,6 +235,7 @@ function DesktopMenu({ item }: { item: string }) {
   const DesktopDropDown = dropDownMapping[item.toLowerCase()];
   const { t } = useTranslation("layout");
   const router = useRouter();
+  const trimmedItem = item.replace(/\s/g, "");
```

Comment:
what character are we removing here? let's a comment

Reply:
removing any spaces! Eg. `Meta chain` to `Metachain`

Reply:
are we standardizing to use `Metachain` across the whole ui? If yes then we can remove the space directly from the source?

Reply:
@nattadex ^

Reply:
nope, the header uses `Meta Chain`

Reply:
does it work if we add a `-`? e.g.: `Meta-Chain`, it is two words anyway, why do we want to combine it?